tests/test_nanoevents_vector.py

```diff
@@ -444,12 +444,18 @@ def test_lorentz_vector_numba(a_dtype, b_dtype):
         110.66616465749593,
         110.68423555321688,
     ]
-    assert pytest.approx(a.delta_phi(b), abs=1e-7) == [
-        0.03369510734601633,
-        -0.1798534997924781,
-        0.33292327383538156,
-        0.6078019961139605,
-    ]
+
+    computed_dphi = a.delta_phi(b).to_numpy()
+
+    assert pytest.approx(computed_dphi, abs=1e-6) == np.array(
```

Comment:
Any idea why the tolerance had to be increased?

Reply:
It needs further investigation but yeah it looks like there's some conversion going on or something that we should investigate. 

This is also related to the above, I was just making sure there wasn't weird roundoff error happening or anything.